Src/ReadGps.c

```diff
@@ -7,14 +7,109 @@
 #include "Data.h"
 
 static int READ_GPS_PERIOD = 1000;
+static int CMD_TIMEOUT = 1500;
+
+static const uint8_t GPS_SET_ANTENNA[] = {0xB5, 0x62, 0x06, 0x13, 0x04, 0x00, 0x1B, 0x00, 0x8B, 0xA9}; // CFG-ANT
+static const uint8_t GPS_SET_DATUM[] = {0xB5, 0x62, 0x06, 0x06, 0x02, 0x00, 0x00, 0x00}; // CFG-DAT
+static const uint8_t GPS_SETUP_UART[];  //TODO: setup configuration for UART [CFG-PRT]
+static const uint8_t GPS_SET_OUTPUT_RATE[] = {0xB5, 0x62, 0x06, 0x08, 0x06, 0x00, 0xFA, 0x00, 0x01, 0x00, 0x01, 0x00}; // CFG-RATE
+static const uint8_t GPS_CONFIG_NMEA[]; // not sure if we need to do manually [CFG_NMEA]
+static const uint8_t GPS_SET_NAVIGATION_MODE[] = {0xB5, 0x62, 0x06, 0x24, 0x24, 0x00, 0xFF, 0xFF, 0x06, 0x03, 0x00, 0x00, 0x00, 0x00, 0x10, 0x27, 0x00, 0x00,
+                                                0x05, 0x00, 0xFA, 0x00, 0xFA, 0x00, 0x64, 0x00, 0x2C, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                                0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x16, 0xDC
+                                               };
+static const uint8_t GPS_test_packet[] = {0111100101, 0110100101};
```

Comment:
unused, and binary can't be written in C this way, needs to be in hex.

Reply:
Ignore that, I had it cause I needed binary numbers that I kept forgetting. I don't have this in my current version of code